src/core/grpc_service.proto

```diff
@@ -65,6 +65,12 @@ service GRPCService
   //@@
   rpc Control(ControlRequest) returns (ControlResponse) {}
 
+  //@@  .. cpp:var:: rpc ShmControl(ShmControlRequest) returns (ShmControlResponse)
```

Comment:
Remove, this file is in the other commit, isn't it?

Reply:
Same

Reply:
Rebased and overwritten.